### DISCUSSION TITLE
FPS questions

### DISCUSSION THREAD

**Discussion by scaruslooner** [Purfview/whisper-standalone-win#217] [Q&A] [answered]
Does fps change the output of srt time? So say for instance, I have two videos that are the same and shot at different fps. (Ex: 23.976 fps and 60 fps) will that effect where the word timestamp will be? Also, if I get SRT from 60 fps video imported into premiere pro timeline/sequence settings set on 23.976 fps will the SRT(captions) be inaccurate?

**Comment by Purfview** [comment, marked as answer]
FPS doesn't matter as srt contains timecode.

**Reply by scaruslooner** [reply]
thanks
